caktus_primer/templates/styleguide/styleguide-modals.html

```diff
@@ -0,0 +1,219 @@
+{% extends "styleguide/styleguide.html" %}
+
+{% load styleguide_tags %}
+<head>
+    {% block extra-css %}
+    <style>
+        dialog {
+            position: relative;
+            padding: 2em 2em;
+            border: none;
+            box-shadow: 2px 3px 4px rgba(0, 0, 0, 0.1);
+        }
+
+        dialog::backdrop {
+            background: #2B2B28;
+            opacity: 0.7;
+        }
+
+        button[aria-label="close"] {
+            position: absolute;
+            height: 44pt; /* Per Apple's HIG we should maintain a tappable area of 44pt by 44pt */
+            width: 44pt;
+            top: 0;
+            right: 0;
+            color: black;
+            border: none;
+            background: transparent;
+            box-shadow: none;
+            font-weight: bold;
+        }
+
+        button[aria-label="close"]:hover {
+            color: red;
+        }
+        
+        @media all and (max-width: 600px) {
+            dialog {
+                width: 90% !important;
+            }
+        }
```

Comment:
This styling needs to be part of `modals.styl` if it is to actually be used by any users of Primer, right?